### PR TITLE
po/doc

### PR DESCRIPTION
**Pull request by TurboGit**
doc for undo/redo and slideshow.

### CODE REVIEW

File: doc/usermanual/lighttable/concepts/undoredo.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <para>
+      Most changes in the lighttable are recorded and can be
+      reverted. This includes modifications in color labels, ratings,
+      geo-localization, tags, metadata, copy/paste of history or
+      applying a style. Thanks to this recording it is possible to
+      undo changes or redo them to recover a previous state. Note that
+      this undo/redo facility is unlimited in the number of steps
+      while in the lighttable but it is reset each time the view is
+      switched (going to the darkroom for example).
+    </para>
```
Comment (brucewilliamsphotography) [inline doc/usermanual/lighttable/concepts/undoredo.xml]: Just added some punctuation....


Most changes in the lighttable are recorded and can be reverted. This includes modifications to color labels, ratings,       geo-localization, tags, metadata, copy/paste of history, or applying a style.
Thanks to this recording, it is possible to undo changes or redo them to recover a previous state. Note that this undo/redo facility is unlimited in the number of steps while in the lighttable, but it is reset each time the view is switched (going to the darkroom for example).

File: doc/usermanual/lighttable/concepts/undoredo.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    <para>
+      The default key shortcuts are:
```
Comment (Balise42) [inline doc/usermanual/lighttable/concepts/undoredo.xml]: I would tend to go for "shortcut keys" or "keyboard shortcuts" - "key shortcuts" doesn't strike me as particularly common, or is it?

File: doc/usermanual/lighttable/concepts/undoredo.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <itemizedlist>
+
+        <listitem><para>
+          <emphasis>Ctrl-z</emphasis> : undo last modifications
```
Comment (Balise42) [inline doc/usermanual/lighttable/concepts/undoredo.xml]: nitpick probably to be ignored, if only for consistency with the rest of the doc: the standard way of tagging that sort of things in DocBook would be <keycombo><keycap>Ctrl</keycap><keycap>Z</keycap></keycombo>.

File: doc/usermanual/slideshow/slideshow.xml [hunk 4]
```diff
@@ -66,15 +66,31 @@
...
+              <entry>
+                <emphasis>-</emphasis> or <emphasis>down</emphasis>
+              </entry>
+              <entry>
+                decrease delay between each image
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <emphasis>left-click</emphasis> or <emphasis>shift-right</emphasis> or <emphasis>right</emphasis>
```
Comment (Balise42) [inline doc/usermanual/slideshow/slideshow.xml]: I think I'd go for "left click or right or shift-right" (same for left below): it seems more logical to me to give the "easier" (single press shortcut vs double press) way first.

Comment (TurboGit) [reply doc/usermanual/slideshow/slideshow.xml]: Done.

File: doc/usermanual/lighttable/concepts/undoredo.xml (hunk 3 above)
Comment (TurboGit) [reply doc/usermanual/lighttable/concepts/undoredo.xml]: Indeed for consistency we need to keep ctrl-z.

File: doc/usermanual/lighttable/concepts/undoredo.xml (hunk 2 above)
Comment (TurboGit) [reply doc/usermanual/lighttable/concepts/undoredo.xml]: Yes, keyboard shortcut seems what is used in doc at the moment. Fixed.

**Comment by TurboGit** [comment]
Manually merged.

File: doc/usermanual/lighttable/concepts/undoredo.xml (hunk 3 above)
Comment (Balise42) [reply doc/usermanual/lighttable/concepts/undoredo.xml]: .... github ate my exact DocBook tagging. Let my try again:
`<keycombo><keycap>Ctrl</keycap><keycap>Z</keycap></keycombo>`.
Anyway. As I was saying, nitpicking :P

**Pull request closed without merging**